packages/sit-onyx/src/components/OnyxNavBar/OnyxNavBar.vue

```diff
@@ -195,7 +194,7 @@ $height: 3.5rem;
       align-items: center;
       gap: $gap;
       height: 100%;
-      padding-inline: var(--onyx-spacing-3xl);
+      padding-inline: var(--onyx-grid-margin);
```

Comment:
question: will this variable be defined even if the nav bar is used without a grid layout surrounding it?

Reply:
Yes that’s a global variable which is always defined